app/models/card.rb

```diff
@@ -3,6 +3,8 @@ class Card < ActiveRecord::Base
   validate  :texts_are_different
   before_validation :set_review_date, on: :create
 
+  scope :to_repeat, -> { where('review_date <= ?', Date.today) }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.